### ISSUE TITLE
Add ability to provide custom/app icon for the status bar notification

### ISSUE THREAD

**Issue by AlienAsRoger** [nishkarsh/android-permissions#33]
Hello! Thanks for the library! 

As a developer, I would like to have the ability to use my own app logo/icon for the status bar notification. Right now it's using the default android icon, which is OK, but I think we can do better.

I believe this part 
```java
 Builder notificationBuilder = (new Builder(this.context, "android-permissions")).setContentTitle(title).setContentText(message).setAutoCancel(true).setPriority(0).setSmallIcon(17629184).setContentIntent(pendingIntent);
```
needs to be updated with an option to use provided icon

Thank you!

**Comment by nishkarsh** [comment]
Hello @AlienAsRoger, thanks for raising this. It, indeed, would be a good functionality to add. We will take a look at it soon.

**Comment by nishkarsh** [comment]
This is released in v2.1.6 and the readme is updated with instructions to customise the notification.
